From d4b61fe94b05419dc991e19562960dee3c19fefa Mon Sep 17 00:00:00 2001
From: Michael Bestas <mikeioannina@gmail.com>
Date: Sat, 17 Aug 2013 13:39:50 +0300
Subject: [PATCH 37/67] Gallery2: Modify AOSP EL translations

Fix wrong gender in plurals

Signed-off-by: Michael Bestas <mikeioannina@gmail.com>

Change-Id: I4fdac15d90f1f37d02029fd88ca404d346c72f3f
(cherry picked from commit 6fffb7036fcfdfd3ae7399db161ea56e5c6427e3)
---
 res/values-el/strings.xml | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/res/values-el/strings.xml b/res/values-el/strings.xml
index 507b21064..0a1be1421 100644
--- a/res/values-el/strings.xml
+++ b/res/values-el/strings.xml
@@ -70,17 +70,17 @@
     <string name="switch_to_camera" msgid="7280111806675169992">"Φωτογραφική μηχανή"</string>
   <plurals name="number_of_items_selected">
     <item quantity="zero" msgid="2142579311530586258">"Επιλέχθηκαν %1$d"</item>
-    <item quantity="one" msgid="2478365152745637768">"Επιλέχθηκαν %1$d"</item>
+    <item quantity="one" msgid="2478365152745637768">"Επιλέχθηκε %1$d"</item>
     <item quantity="other" msgid="754722656147810487">"Επιλέχθηκαν %1$d"</item>
   </plurals>
   <plurals name="number_of_albums_selected">
     <item quantity="zero" msgid="749292746814788132">"Επιλέχθηκαν %1$d"</item>
-    <item quantity="one" msgid="6184377003099987825">"Επιλέχθηκαν %1$d"</item>
+    <item quantity="one" msgid="6184377003099987825">"Επιλέχθηκε %1$d"</item>
     <item quantity="other" msgid="53105607141906130">"Επιλέχθηκαν %1$d"</item>
   </plurals>
   <plurals name="number_of_groups_selected">
     <item quantity="zero" msgid="3466388370310869238">"Επιλέχθηκαν %1$d"</item>
-    <item quantity="one" msgid="5030162638216034260">"Επιλέχθηκαν %1$d"</item>
+    <item quantity="one" msgid="5030162638216034260">"Επιλέχθηκε %1$d"</item>
     <item quantity="other" msgid="3512041363942842738">"Επιλέχθηκαν %1$d"</item>
   </plurals>
     <string name="show_on_map" msgid="6157544221201750980">"Εμφάνιση στον χάρτη"</string>
-- 
2.17.1

